################################################################################
SHORT DESCRIPTION: 
################################################################################
Remove inline function declarations.

################################################################################
LONG DESCRIPTION: 
################################################################################
These functions are used in different translation units and hence cannot be
declared inline.

################################################################################
CHANGELOG 
################################################################################
Author(s):
Mahantesh Salimath <salimathm@ainfosec.com>

################################################################################
REMOVAL 
################################################################################

################################################################################
UPSTREAM PLAN
################################################################################
Yes, should be upstreamed.

################################################################################
INTERNAL DEPENDENCIES 
################################################################################

################################################################################
PATCHES 
################################################################################
--- a/drivers/tapdisk-utils.h
+++ b/drivers/tapdisk-utils.h
@@ -100,6 +100,6 @@ shm_create(struct shm *shm);
 int
 shm_destroy(struct shm *shm);
 
-inline long long timeval_to_us(struct timeval *tv);
+long long timeval_to_us(struct timeval *tv);
 
 #endif
--- a/drivers/tapdisk-vbd.h
+++ b/drivers/tapdisk-vbd.h
@@ -241,5 +241,5 @@ void tapdisk_vbd_stats(td_vbd_t *, td_st
 /**
  * Tells whether the VBD contains at least one dead ring.
  */
-bool inline tapdisk_vbd_contains_dead_rings(td_vbd_t * vbd);
+bool tapdisk_vbd_contains_dead_rings(td_vbd_t * vbd);
 #endif
--- a/drivers/td-blkif.h
+++ b/drivers/td-blkif.h
@@ -268,21 +268,21 @@ tapdisk_xenblkif_find(const domid_t domi
  * channel can be shared by multiple block interfaces, the event ID will be
  * shared as well.
  */
-extern inline event_id_t
+extern event_id_t
 tapdisk_xenblkif_evtchn_event_id(const struct td_xenblkif *blkif);
 
 /**
  * Returns the event ID associated wit checking the ring. This is a private
  * event.
  */
-extern inline event_id_t
+extern event_id_t
 tapdisk_xenblkif_chkrng_event_id(const struct td_xenblkif * const blkif);
 
 /**
  * Returns the event ID associated with stopping polling. This is a private
  * event.
  */
-extern inline event_id_t
+extern event_id_t
 tapdisk_xenblkif_stoppolling_event_id(const struct td_xenblkif * const blkif);
 
 /**
